internal/promotion/runner/builtin/git_cloner_test.go

```diff
@@ -115,6 +115,57 @@ func Test_gitCloner_validate(t *testing.T) {
 				"checkout.0: Must validate one and only one schema",
 			},
 		},
+		{
+			name: "signingKey is provided and valid",
```

Comment:
You probably don't need a separate test case for this. For things that _should be_ valid, you can just add them to the "kitchen sink" test case.